### PR TITLE
Feature/add sdk documentation

### PR DESCRIPTION
**Pull request by LrxGaelle**

### CODE REVIEW

File: src/viewer/viewer_sdk.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+### Compiles and hot-reloads for development
+
+```bash
+npm run dev
+```
+
+### Usage
+
+When going on [http://localhost:8080](http://localhost:8080), a simple interface will parse your projects and models and let you open the one you want.
```
Comment (Kurtil) [inline src/viewer/viewer_sdk.md]: => a simple interface will let you open the project and model you want.

File: src/.vuepress/components/Home.vue [hunk 2]
```diff
@@ -35,7 +33,7 @@
         <!-- <BIMDataCard>
```
Comment (NicolasRichel) [inline src/.vuepress/components/Home.vue]: You got some commented (dead?) code here.
Maybe it should be removed.

File: src/viewer/viewer_sdk.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+You can directly open one by opening an URL using specific Ids, for example : [http://localhost:8080/viewer?cloudId=391&projectId=634&ifcId=1491](http://localhost:8080/viewer?cloudId=391&projectId=634&ifcId=1491)
+
+## Create your plugin
+
+```bash
+npm run init-plugin
+```
+
+This tool asks you a couple of questions about the plugin you develop and generate from your answers basic files for your plugin.
```
Comment (Kurtil) [inline src/viewer/viewer_sdk.md]: basic => boilerplate ?

**Review by Kurtil — CHANGES_REQUESTED** [review]

File: src/viewer/viewer_sdk.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
+# Viewer SDK
+
+We have pre-configured a **VueJS** environment to develop BIMData Viewer plugins. You can develop, test, build, package and share your plugin easily. 
+
+In this section, you get tutorials and references for installing and using the [BIMData Viewer SDK](https://github.com/bimdata/bimdata-viewer-sdk)
+
+## Setup
+
+First, you have to clone the sdk repo and install SDK dependecies :
```
Comment (NicolasRichel) [inline src/viewer/viewer_sdk.md]: Typo here: **dependecies** instead of **dependencies**.

**Review by NicolasRichel — CHANGES_REQUESTED** [review]

**Review by Kurtil — APPROVED** [review]

**Review by NicolasRichel — APPROVED** [review]

**Review by Amoki — APPROVED** [review]

**Review by amineau — APPROVED** [review]

**Pull request merged**